msp/mspimpl.go

```diff
@@ -890,9 +890,15 @@ func (msp *bccspmsp) getCertificationChainIdentifierFromChain(chain []*x509.Cert
 // is regenerated to have a Low-S signature.
 func (msp *bccspmsp) sanitizeCert(cert *x509.Certificate) (*x509.Certificate, error) {
 	if isECDSASignedCert(cert) {
+		validityOpts := msp.getValidityOptsForCert(cert)
+		if bytes.Equal(cert.RawSubject, cert.RawIssuer) && cert.IsCA {
```

Comment:
this only affects self signed root CAs, no? What about intermediate certs that have parents that are intermediate?

Reply:
Yes, there is no problem with intermediates